IdentityCore/src/oauth2/token/MSIDBaseToken.m

```diff
@@ -101,6 +101,11 @@ - (MSIDTokenType)tokenType
     return MSIDTokenTypeOther;
 }
 
+- (BOOL)supportsTokenType:(MSIDTokenType)tokenType
+{
+    return tokenType == MSIDTokenTypeOther || tokenType == self.tokenType;
```

Comment:
What is the case when we need to support MSIDTokenTypeOther?

Reply:
Fixed